### PR TITLE
Show token comments and font families in docs

### PR DESCRIPTION
**Pull request by tylersticka**
## Overview

Makes several improvements to our Theo design token translation task:

- If any token in a category includes comments, they will be displayed in a new "comment" column.
- If a token is in category `font-family`, the value will have each step of the stack applied for preview purposes.
- All token names and non-font-family token values are wrapped in `<code>` so they are more consistent line to line.

## Screenshots

Screen Shot 2020-04-14 at 4 33 48 PM

Screen Shot 2020-04-14 at 4 33 59 PM

### CODE REVIEW

File: .theo/formats/stories.mdx.js [hunk 1]
```diff
@@ -54,12 +79,15 @@ function mdxProp(prop) {
  * @returns {String}
  */
 function mdxProps(props) {
-  const rows = props.map(mdxProp);
+  const commentedProps = props.filter((prop) => !!prop.comment);
```
Comment (Josh68) [inline .theo/formats/stories.mdx.js:82]: Nit. I doubt this matters, but `'comment' in props` would return `true` for props with an empty comment, while this code wouldn't. But what you have is probably the behavior you want.

Comment (tylersticka) [reply .theo/formats/stories.mdx.js:82]: Yeah, I don't want it to return true for an empty comment.

Is this blocking approval?

File: .theo/formats/stories.mdx.js [hunk 2]
```diff
@@ -36,15 +36,40 @@ function mdxColors(props) {
...
+    content += `<td>${fontStrings.join(', ')}</td>`;
+  } else {
+    content += `<td><code>${prop.value}</code></td>`;
+  }
+
+  if (includeComment) {
+    content += `<td>${prop.comment || '&nbsp;'}</td>`;
+  }
+
+  return `<tr>${content.trim()}</tr>`;
```
Comment (Josh68) [inline .theo/formats/stories.mdx.js:72]: Not sure, but is there any need to add `.trim()` back after the `return`ed `<tr>`-wrapped string?

File: .theo/formats/stories.mdx.js (hunk 1 above)
Comment (Josh68) [reply .theo/formats/stories.mdx.js:82]: No, approved. Just trying to be thorough.

Comment (spaceninja) [reply .theo/formats/stories.mdx.js:82]: Can confirm this is working as intended — to filter the props array and return only props with comments.

File: .theo/formats/stories.mdx.js (hunk 2 above)
Comment (Josh68) [reply .theo/formats/stories.mdx.js:72]: On second look, there is no need for that. Resolving this comment

**Pull request merged**